executor/gen.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2017 syzkaller project authors. All rights reserved.
 // Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
-// +build !openbsd !netbsd
+// +build !freebsd,!openbsd,!netbsd
```

Comment:
Right. `,` is AND, ` ` is OR.
https://golang.org/pkg/go/build/#hdr-Build_Constraints